test/cpp_extensions/cuda_extension_kernel.cu

```diff
@@ -0,0 +1,27 @@
+#include <cuda.h>
+#include <cuda_runtime.h>
+#include <stdio.h>
+
+__global__ void sigmoid_add_kernel(
+    const float* __restrict__ x,
+    const float* __restrict__ y,
+    float* __restrict__ output,
+    const int size) {
+  const int index = blockIdx.x * blockDim.x + threadIdx.x;
+  if (index < size) {
+    const float sigmoid_x = 1.0f / (1.0f + __expf(-x[index]));
+    const float sigmoid_y = 1.0f / (1.0f + __expf(-y[index]));
+    output[index] = sigmoid_x + sigmoid_y;
+  }
+}
+
+void sigmoid_add_cuda(const float* x, const float* y, float* output, int size) {
+  const int threads = 1024;
+  const int blocks = (size + threads - 1) / threads;
+  printf(
+      "Launching kernel with %d block(s) x %d thread(s) for %d values\n",
+      blocks,
+      threads,
+      size);
```

Comment:
Is this meant to stay here? Looks like it's a leftover after some debugging

Reply:
It was a conscious choice, just for informative output. But I can get rid of it.